### PR TITLE
complete block list with loomSDK

### PR DESCRIPTION
**Pull request by lingwebber**

### CODE REVIEW

**Review by enlight — CHANGES_REQUESTED** [review]
Run `yarn format` to get rid of irrelevant code style changes.

File: src/blockchain.ts [hunk 1]
```diff
@@ -71,6 +74,7 @@ export class Blockchain {
 
   setServerUrl(newUrl: string) {
     if (this.serverUrl !== newUrl) {
+      console.log('set Server Url')
```
Comment (enlight) [inline src/blockchain.ts]: Is this needed?

File: src/components/card.ts [hunk 2]
```diff
@@ -1,24 +1,21 @@
...
 export default class Card extends Vue {
-
-  @Prop() elementId!: string
+  @Prop() elementId: string
 
   mounted() {
-    console.log("The prop: ", this.elementId)
-    this.createChart(this.elementId, planetChartData);
-  } 
+    console.log('The prop: ', this.elementId)
```
Comment (enlight) [inline src/components/card.ts]: And this?

File: src/components/nav.ts [hunk 3]
```diff
@@ -3,6 +3,6 @@ import { Component, Prop, Vue } from 'vue-property-decorator'
 @Component({})
 export default class Nav extends Vue {
   mounted() {
-    console.log("Nav is mounted")
+    console.log('Nav is mounted')
```
Comment (enlight) [inline src/components/nav.ts]: Or this?

**Comment by enlight** [comment]
I just noticed this PR isn't against `master`, did you mean to merge it into the `redesign` branch?

**Comment by lingwebber** [comment]
Yeah. It's based on redesign branch and I thought it should be merged into redesign..

**Pull request merged**